migrations/20201123194206-create-events-users.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('Events_Users', {
+      id: {
+        type: Sequelize.INTEGER,
+        allowNull: false,
+        primaryKey: true,
+        autoIncrement: true
+      },
+      userId: {
+        type: Sequelize.INTEGER,
+        allowNull: false
+      },
+      eventId: {
+        type: Sequelize.INTEGER,
+        allowNull: false
+      },
+    });
+  },
+  down: async (queryInterface, Sequelize) => {
+    await queryInterface.dropTable('Events_Users');
+  }
+};
```

Comment:
You can use one migration for one usecase. However, you need to wrap all steps into one transaction.
Here you have one usecase like 'initiate db', so this could be written in 1 migration file.